starters/lit-mobx-vaadin/src/pages/greeting/greeting.ts

```diff
@@ -0,0 +1,69 @@
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { html, PropertyValueMap } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { BeforeEnterObserver, RouterLocation } from '@vaadin/router';
+import { fetchMessage } from '../../api/fetch.js';
+import { starterState } from '../../state.js';
+import { PageMixin } from '../page.mixin.js';
+
+import '../../components/greeting/greeting.js';
+import '../../components/error-message/error-message.js';
```

Comment:
that's how module resolution is setup in this project

> relative import paths need full extensions (e.g we have to write import "./foo.js" instead of import "./foo")